packages/store/src/plugin-manager.ts

```diff
@@ -1,8 +1,9 @@
-import { Injectable, Optional, SkipSelf, Inject } from '@angular/core';
-import { NgxsPluginFn, NGXS_PLUGINS, NgxsPlugin } from './symbols';
+import { Inject, Injectable, Optional, SkipSelf } from '@angular/core';
+import { NGXS_PLUGINS, NgxsPlugin, NgxsPluginFn } from './symbols';
 
 /**
  * Plugin manager class
+ * @publicApi
```

Comment:
Max, could you remove these comments? I mean `@publicApi` and `@ignore`. This is for `typedoc`, I've seen a PR where you remove it.

Reply:
now it became clear why we wrote `@ignore`

Reply:
done